tests/components/logbook/test_init.py

```diff
@@ -711,23 +706,25 @@ async def test_logbook_entity_filter_with_automations(
 
     # Test entries for 3 days with filter by entity_id
     end_time = start + timedelta(hours=72)
-    response = await client.get(
-        f"/api/logbook/{start_date.isoformat()}?end_time={end_time}&entity=alarm_control_panel.area_001"
+    query = urlencode(
+        {"end_time": end_time.isoformat(), "entity": "alarm_control_panel.area_001"}
     )
+    response = await client.get(f"/api/logbook/{start_date.isoformat()}?{query}")
     assert response.status == HTTPStatus.OK
     json_dict = await response.json()
     assert len(json_dict) == 1
     assert json_dict[0]["entity_id"] == entity_id_test
 
     # Tomorrow time 00:00:00
     start = dt_util.utcnow()
-    start_date = datetime(start.year, start.month, start.day)
+    start_date = datetime(start.year, start.month, start.day, tzinfo=dt_util.UTC)
 
     # Test entries from today to 3 days with filter by entity_id
     end_time = start_date + timedelta(hours=72)
-    response = await client.get(
-        f"/api/logbook/{start_date.isoformat()}?end_time={end_time}&entity=alarm_control_panel.area_002"
+    query = urlencode(
+        {"end_time": end_time.isoformat(), "entity": "alarm_control_panel.area_002"}
     )
+    response = await client.get(f"/api/logbook/{start_date.isoformat()}?{query}")
```

Comment:
Shouldn't we just use [the `params` keyword arg](https://docs.aiohttp.org/en/stable/client_quickstart.html#passing-parameters-in-urls)?

Reply:
I _think_ that will work. I've been burned with aiohttp's encoder before around `+` and ` ` so my first instinct is to stay away.

Will test and adjust

Reply:
29a3f6d7ba23cf6a3e5f33035a95368598a464a0